README.md

```diff
@@ -100,6 +101,14 @@ dependencies {
   compile "org.ice1000.jimgui:core:$jimgui_version" // basic functionality
   compile "org.ice1000.jimgui:kotlin-dsl:$jimgui_version" // kotlin dsl wrapper
 }
+// ...
+run {
+  if (OperatingSystem.current() == OperatingSystem.MAC_OS) {
+    // cocoa requires glfw running in main thread
+    jvmArgs "-XstartOnFirstThread"
```

Comment:
Can we separate this part as something like a "notice" block, which inform Mac users to add this piece of code directly?